sanic/router.py

```diff
@@ -23,6 +23,10 @@ class RouteExists(Exception):
     pass
 
 
+class RouteDontExists(Exception):
```

Comment:
Can we change this to `RouteDoesNotExist`? I think it works better in telling what has happened. Or even a `RouteNotFound` exception would suffice in this case.